recipes/entityx/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+required_conan_version = ">=1.33.0"
+
+
+class EntityXConan(ConanFile):
+    name = "entityx"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/alecthomas/entityx/"
+    topics = ("entity", "conan")
+    license = "MIT"
+    description = "EntityX is an EC system that uses C++11 features to provide type-safe component management, event delivery, etc."
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    generators = "cmake"
+
+    exports_sources = "CMakeLists.txt"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("entityx-" + self.version, self._source_subfolder)
+
+    def validate(self):
+        if self.settings.os == "Windows" and self.options.shared:
+            raise ConanInvalidConfiguration("Windows shared library do not export all symbols")
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["ENTITYX_BUILD_SHARED"] = "1" if self.options.shared else "0"
+        self._cmake.definitions["ENTITYX_BUILD_TESTING"] = "0"
```

Comment:
```suggestion
        self._cmake.definitions["ENTITYX_BUILD_SHARED"] = self.options.shared
        self._cmake.definitions["ENTITYX_BUILD_TESTING"] = False
        self._cmake.definitions["ENTITYX_RUN_BENCHMARKS"] = False
```

There also 2 options that we would like to add in recipe:
https://github.com/alecthomas/entityx/blob/1.3.0/CMakeLists.txt#L23-L24